assets/app/actions/createDraftBranchName.js

```diff
@@ -0,0 +1,5 @@
+import { encodeB64 } from '../util/encoding';
+
+export default path => {
```

Comment:
If we can make a logical grouping like this, I'm all for it. We can always regroup/rename later as the design evolves.